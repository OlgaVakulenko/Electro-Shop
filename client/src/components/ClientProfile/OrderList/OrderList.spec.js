import React from 'react';
import { shallow } from 'enzyme';

import { OrderList } from './OrderList';

describe('The test of OrderList component', () => {
  it('OrderList was rendered', () => {
    const rendered = shallow(
      <OrderList />,
    );

    expect(rendered).toMatchSnapshot();
  });
});