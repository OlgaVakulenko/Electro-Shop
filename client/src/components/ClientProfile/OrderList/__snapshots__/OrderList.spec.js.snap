// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The test of OrderList component OrderList was rendered 1`] = `
<Fragment>
  <WithStyles(ForwardRef(ExpansionPanel))
    expanded={false}
    onChange={[Function]}
  >
    <WithStyles(ForwardRef(ExpansionPanelSummary))
      aria-controls="panel2bh-content"
      expandIcon={<Memo(ExpandMoreIcon) />}
      id="panel2bh-header"
    >
      <WithStyles(ForwardRef(Typography))
        className="makeStyles-heading-2"
        style={
          Object {
            "color": "#6A86E8",
          }
        }
      >
        My Order List
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(ExpansionPanelSummary))>
    <WithStyles(ForwardRef(ExpansionPanelDetails))
      className="makeStyles-expansionPanelDetails-1"
    >
      <WithStyles(ForwardRef(Typography))>
        Donec placerat, lectus sed mattis semper, neque lectus feugiat lectus, varius pulvinar diam eros in elit. Pellentesque convallis laoreet laoreet.
      </WithStyles(ForwardRef(Typography))>
    </WithStyles(ForwardRef(ExpansionPanelDetails))>
  </WithStyles(ForwardRef(ExpansionPanel))>
</Fragment>
`;
